index.html

```diff
@@ -2497,10 +2505,12 @@
         The set of presentations known to the user agent should be cleared when
         the user requests to "clear browsing data."
       </p>
-      <div class="issue">
-        The spec should clarify what is to happen to the set of known
-        presentations in "incognito" (private browsing context) mode.
-      </div>
+      <p>
+        When in private browsing mode ("incognito"), the set of presentations
+        known to the user agent must be initially empty, and all state
```

Comment:
Updated the PR to clarify that this applies to the controlling user agent.

I think stating that there are no presentation connections inherited from the existing (non-incognito) session is the right assertion.  If the user explicitly creates a connection via `start()`, or initiating a `defaultRequest`, that should be possible.  Such connections should go away when the session ends.

Updated the wording to try to capture this.